packages/react/src/components/Provider/Provider.tsx

```diff
@@ -146,41 +160,49 @@ class Provider extends React.Component<WithAsProp<ProviderProps>> {
   }
 
   render() {
-    const { as, theme, variables, children, ...unhandledProps } = this.props
-
+    const {
+      as,
+      theme,
+      rtl,
+      disableAnimations,
+      renderer,
+      variables,
+      children,
+      ...unhandledProps
+    } = this.props
+    const context: ProviderContextInput = {
+      theme,
+      rtl,
+      disableAnimations,
+      renderer,
+    }
     // rehydration disabled to avoid leaking styles between renderers
     // https://github.com/rofrischmann/fela/blob/master/docs/api/fela-dom/rehydrate.md
+    const outgoingContext: ProviderContextPrepared = mergeContexts(this.context, context)
+
+    // Heads up!
+    // We should call render() to ensure that a subscription for DOM updates was created
+    // https://github.com/stardust-ui/react/issues/581
+    if (isBrowser()) render(outgoingContext.renderer)
```

Comment:
lets import this function as something like `felaDomRender`, just to disambiguate from lots of other renders defined here